blocklylib-core/src/main/java/com/google/blockly/android/ui/BlockViewFactory.java

```diff
@@ -278,6 +278,7 @@ protected FieldView buildFieldView(Field field) {
             case Field.TYPE_ANGLE: {
                 BasicFieldAngleView fieldAngleView = new BasicFieldAngleView(mContext);
                 fieldAngleView.setField((FieldAngle) field);
+                fieldAngleView.setWorkspaceHelper(mHelper);
```

Comment:
This won't be necessary if we use resource strings, as mentioned [here](https://github.com/google/blockly-android/pull/453#issuecomment-266221013).